lib/src/panel.dart

```diff
@@ -255,170 +274,165 @@ class _SlidingUpPanelState extends State<SlidingUpPanel>
 
   @override
   Widget build(BuildContext context) {
+    
     return Stack(
       alignment: widget.slideDirection == SlideDirection.UP
           ? Alignment.bottomCenter
           : Alignment.topCenter,
       children: <Widget>[
         //make the back widget take up the entire back side
         widget.body != null
-            ? AnimatedBuilder(
-                animation: _ac,
-                builder: (context, child) {
-                  return Positioned(
-                    top: widget.parallaxEnabled ? _getParallax() : 0.0,
-                    child: child ?? SizedBox(),
-                  );
```

Comment:
What happened to the parallax?

Reply:
Thanks for contributing! Looks like this breaks parallax behaviors.

Reply:
Since I believed you had abandoned the project, I removed the parallax for my own needs. Merge the first three commits :D

Reply:
Not my project! Though we should have maintained version. Might put one up